setup.py

```diff
@@ -3,9 +3,6 @@
 import email.utils
 import os
 import re
-import sys
-
-from setuptools import setup
 
 from setuptools import setup
```

Comment:
Oh wow how.  Note to self: GitHub's web UI for conflict resolution not actually good enough ;)

Reply:
Yeah, PyCharms is much better 😉